.circleci/config.yml

```diff
@@ -166,9 +167,9 @@ workflows:
           parallel: true
           parallelism: 2
           group: 'care ops chrome'
-          start: 'npm start'
```

Comment:
This was a guess. But I figured since the custom command is calling `npm start`, we didn't need this. Or we could leave this and remove the the `npm start` from the command